src/core/style/expressions/quantiles.js

```diff
@@ -2,13 +2,14 @@ import Expression from './expression';
 import { float } from '../functions';
 import { checkNumber, checkInstance, checkType } from './utils';
 import Property from './property';
+import * as schema from '../../schema';
 
 let quantilesUID = 0;
 
 function genQuantiles(global) {
     return class Quantiles extends Expression {
         constructor(input, buckets) {
-            checkInstance('quantiles', 'input', 0, Property, input);
+            checkInstance('quantiles', 'input', 0, Property, input && (input.property || input));
```

Comment:
Clarify, pls

Reply:
This is added to allow aggregation as input parameters for quantiles. Ex `quantiles(sum($prop), 5)`. In this case we should check input.property instead of input.